script/Ciel/Day 15/15_01A_CIEL15A_1.txt

```diff
@@ -1853,13 +1861,14 @@ My eyes fill with tears at my own words.
 [sha:9c60229a6f2b4db86df8cb81dfe555c777e2ff06]{
 -- Page 114, Offset 41723.
 -- ―――気がつかなければよかった。
-―――I shouldn't have noticed.
+I shouldn't have noticed.
 }
 [sha:f67242ba0d76c58992e31b00b6db9ad25fa1bc17]{
 -- Page 114, Offset 41724.
 -- そうすれば―――このまま幸せな時間が、誰も傷つくことのない一日が続いていただろうから。
 Because then this beautiful moment――
-this wonderful day without any troubles would surely have kept on going.
+this wonderful day without any troubles――
+would surely have kept on going.
```

Comment:
preference to change 'kept on going' to 'continued'

Reply:
Feels a bit short. Maybe invert it so that it works better with both moment and day?
> Because then this beautiful moment――
> this wonderful day without any troubles――
> wouldn't have had to end.

Reply:
I think the original is preferrable